src/KNNClassifier/index.js

```diff
@@ -162,29 +164,37 @@ class KNN {
     await io.saveBlob(JSON.stringify({ dataset, tensors }), fileName, 'application/octet-stream');
   }
 
-  load(path, callback) {
-    io.loadFile(path, (err, data) => {
-      if (data) {
-        const { dataset, tensors } = data;
-        this.mapStringToIndex = Object.keys(dataset).map(key => dataset[key].label);
-        const tensorsData = tensors
-          .map((tensor, i) => {
-            if (tensor) {
-              const values = Object.keys(tensor).map(v => tensor[v]);
-              return tf.tensor(values, dataset[i].shape, dataset[i].dtype);
-            }
-            return null;
-          })
-          .reduce((acc, cur, j) => {
-            acc[j] = cur;
-            return acc;
-          }, {});
-        this.knnClassifier.setClassifierDataset(tensorsData);
-        if (callback) {
-          callback();
-        }
+  load(pathOrData, callback) {
+    if (typeof pathOrData === 'object') {
+      this[loadData](pathOrData, callback);
```

Comment:
Should this be `this.loadData(pathOrData, callback)`?

Reply:
```suggestion
      this.loadData(pathOrData, callback);
```

Reply:
If I define loadData in a standard way, both "load" and "loadData" becomes accessible.

<img width="340" alt="2019-02-21 13 01 46" src="https://user-images.githubusercontent.com/10215/53142937-37b07f80-35d9-11e9-8423-61f218eed299.png">

I thought that the intention of merging these two is to make "load" accessible and to hide "loadData" from the user.